diff -ruN radvd-0.7.2.orig/radvd.conf.5.man radvd-0.7.2/radvd.conf.5.man
--- radvd-0.7.2.orig/radvd.conf.5.man	2002-06-21 13:57:15.000000000 +0200
+++ radvd-0.7.2/radvd.conf.5.man	2004-01-24 15:49:30.000000000 +0100
@@ -290,7 +290,7 @@
 represents infinity (i.e. a value of all one bits (0xffffffff)).
 The valid lifetime is also used by RFC 2462.
 
-Default: infinity
+Default: 2592000 seconds (30 days)
 
 .TP
 .BR "AdvPreferredLifetime " seconds "" | infinity
